circle.yml

```diff
@@ -1,6 +1,6 @@
 machine:
   node:
-    version: 6.2.2
+    version: 8.2.1
```

Comment:
Does this change need to be a part of this PR?

If not, can you factor it out? It's easier to deal with 2 smaller independent PRs.